week08_pomdp/practice_pytorch.ipynb

```diff
@@ -141,7 +134,7 @@
     "\n",
     "Let's design another agent that has a recurrent neural net memory to solve this. Here's a sketch.\n",
     "\n",
-    "![img](img1.jpg)\n"
+    "![img](https://github.com/yandexdataschool/Practical_RL/blob/master/week08_pomdp/img1.jpg?raw=1)\n"
```

Comment:
reviewnb complains about this: the image is not rendered at all, while previously it worked fine. I'd guess a different link such as https://raw.githubusercontent.com/yandexdataschool/Practical_RL/master/week08_pomdp/img1.jpg is needed here.

Reply:
Weirdly, I can see it works fine!!!

![image](https://user-images.githubusercontent.com/72295771/205714221-2fff31b2-830e-4d10-b090-40b051770ce2.png)